web/src/user_group_edit.js

```diff
@@ -64,31 +64,38 @@ export function get_edit_container(group) {
     );
 }
 
-function update_add_members_elements(group) {
+function update_group_settings_and_add_members_elements(group) {
     if (!is_editing_group(group.id)) {
         return;
     }
 
-    // We are only concerned with the Members tab for editing groups.
+    // We are concerned with the Members tab for editing groups.
     const $add_members_container = $(".edit_members_for_user_group .add_members_container");
 
+    // We are concerend with the General tab for changing group permissions.
+    const $group_permission_settings = $("#group_permission_settings");
```

Comment:
I thought like since this would be the case with all of the permission dropdown elements that would be added in future, I would get the container of these dropdowns as `$group_permission_settings` and then get the dropdown elements there using `$group_permission_settings.find(".dropdown-widget-button");`.

So in future if we add more dropdowns, we just need to iterate on `$permission_dropdown_element` and disable them.

Reply:
Sounds good.